Lyft case study (#457)
sourcegraph/about: blogposts/production-stability-at-lyft-during-their-monolith-to-microservices-decomposition.md

---
title: How Sourcegraph ensured production stability at Lyft during their monolith to microservices decomposition
author: Ryan Blunden
publishDate: 2019-12-13T10:00-07:00
tags: [
  blog
]
slug: production-stability-at-lyft-during-their-monolith-to-microservices-decomposition
heroImage: https://about.sourcegraph.com/blog/lyft-logo.png
published: true

> description: "Learn how Sourcegraph code search helped Lyft ensure (largely) issue free production deploys during their monolith to microservices decomposition"

Reviewer: ```suggestion
description: "Learn how Sourcegraph code search helped Lyft ensure (largely) issue-free production deploys during their monolith to microservices decomposition"
```

---

<!-- pending approval 
<p style="text-align: center">
  <img src="/case-studies/lyft-sourcegraph-case-study-og-embed.jpg" />
</p> -->

> Lyft's mission is to improve people's lives with the world's best transportation. With over 30 million riders in 2018, Lyft's business is growing, as is the number of engineers, repositories, and microservices.

Reviewer: ```suggestion
Lyft's mission is to improve people's lives with the world's best transportation. With over 30 million riders in 2018, Lyft's business is growing, as are the numbers of engineers, repositories, and microservices.
```

> Learn how, Sourcegraph's [multi-repository code search](/product/code-search-navigation) helped Lyft ensure (largely) issue free production deploys during their monolith to microservices decomposition.

Reviewer: ```suggestion
Learn how Sourcegraph's [multi-repository code search](/product/code-search-navigation) helped Lyft ensure (largely) issue-free production deploys during their monolith to microservices decomposition.
```
